### ISSUE TITLE
Dashpages UX improvements

### ISSUE THREAD

**Issue by samsp-msft** [dotnet/aspire#5347]
Looking at the feature so far, I think we should make some subtle UX changes to the implementation:

- [ ] Create a separate logical section for DashPages from All Metrics
![image](https://github.com/user-attachments/assets/34c954fc-4125-4bfc-bc5f-76c259605a99)

- [ ] Remove the description field from each metric and reduce the amount of vertical space in the title of each metric panel

- [ ] Replace the filter panel on each metric with a dropdown in the title bar of each metric view.
![image](https://github.com/user-attachments/assets/58470394-7378-444b-bac0-65cef15981b8)
Clicking the button will show either a popup or side panel with the filter UI for controlling the view.

Q: If you change the filter view for a dashpage - how long does that persist for? If you switch dashpage and come back are the filters still applied? If you switch to another resource will that have the same view. I think today that switching away will reset the views. We should have a decision on the level of persistence, and have an ability to reset the view.

**Comment by samsp-msft** [comment]
@drewnoakes FYI

@leslierichardson95 we should have a review with UX team pretty soon

@JamesNK - do you have thoughts about the filtering persistence?

**Comment by leslierichardson95** [comment]
> @drewnoakes FYI
> 
> @leslierichardson95 we should have a review with UX team pretty soon
> 
> @JamesNK - do you have thoughts about the filtering persistence?

@drewnoakes @samsp-msft @jamesNK If all or some of you are free, there's a UX office hour we can go to on Thursday .

**Comment by adamint** [comment]
> Q: If you change the filter view for a dashpage - how long does that persist for? If you switch dashpage and come back are the filters still applied? If you switch to another resource will that have the same view. I think today that switching away will reset the views. We should have a decision on the level of persistence, and have an ability to reset the view.

No, the filters will not still be applied after switching dashpages. Any decision on persistence must take into consideration single metric pages as well.

**Comment by adamint** [comment]
@samsp-msft I see that you kept the padding to the left of the individual dashpages to keep parity with the rest of the tree-view, but I would suggest that be removed, as the dashpages are top-level items, not children, here.

**Comment by drewnoakes** [comment]
> Create a separate logical section for DashPages from All Metrics

I don't think the extra level of navigation here adds much. And when no dashpages are configured (or none are available due to missing metrics), the extra level will be redundant.

> Remove the description field from each metric and reduce the amount of vertical space in the title of each metric panel

This is being tracked in #5287.

> Replace the filter panel on each metric with a dropdown in the title bar of each metric view.

Also tracked in #5287.

> how long does that persist for?

Can we make an issue that's just tracking filter persistence please? That's a new feature.
